conf/app.routes

```diff
@@ -1,2 +1,3 @@
 POST         /movements/arrivals/:arrivalId/messages                 controllers.ArrivalTestMessagesController.injectEISResponse(arrivalId: ArrivalId)
-POST         /movements/departures/:departureId/messages             controllers.DepartureTestMessagesController.injectEISResponse(departureId: DepartureId)
\ No newline at end of file
+POST         /movements/departures/:departureId/messages             controllers.DepartureTestMessagesController.injectEISResponse(departureId: DepartureId)
+POST         /movements/departures                                   controllers.DepartureTestMessagesController.submitDepartureDeclaration
```

Comment:
If we're adding the POST requests to the API we're going to start to get confusing behaviour. Especially if we simply mimic the original backend route names. For example the existing routes point to the NCTSControllers for the services, but its understandable that someone will think that we can send other messages via those endpoints.

Reply:
Is the microservice "common-transit-convention-traders-test-support" is for internal testing right correct me if I am wrong.

Reply:
Strictly it was designed for enabling self-service of the NCTS endpoints, BUT now that its not deployed, thats correct

Reply:
Is it going to be deployed to PROD anytime in the future or it is only used for testing.

Reply:
I don't expect so. I very much hope not as it'll mean a lot of cleanup